pkg/network/usm/monitor_tls_test.go

```diff
@@ -780,3 +781,73 @@ func getHTTPLikeProtocolStats(monitor *Monitor, protocolType protocols.ProtocolT
 	}
 	return res
 }
+
+func (s *tlsSuite) TestNodeJSTLS() {
+	const (
+		expectedOccurrences = 10
+		serverPort          = "4444"
+	)
+
+	t := s.T()
+
+	cert, key, err := testutil.GetCertsPaths()
+	require.NoError(t, err)
+
+	require.NoError(t, nodejs.RunServerNodeJS(t, key, cert, serverPort))
+	nodeJSPID, err := nodejs.GetNodeJSDockerPID()
+	require.NoError(t, err)
+
+	cfg := config.New()
+	cfg.EnableHTTPMonitoring = true
+	cfg.EnableNodeJSMonitoring = true
+	cfg.BPFDebug = true
```

Comment:
I think we can disable eBPF debugging, like in our other tests.

Reply:
oops, good catch